scheduler/generic_sched.go

```diff
@@ -577,6 +577,7 @@ func (s *GenericScheduler) computePlacements(
 			// Store the available nodes by datacenter
 			s.ctx.Metrics().NodesAvailable = byDC
 			s.ctx.Metrics().NodesInPool = len(nodes)
+			s.ctx.Metrics().NodePool = s.job.NodePool
```

Comment:
I think we need this in the system scheduler too around here: https://github.com/hashicorp/nomad/blob/e13ceab85553c72d0b17dfe09d2aa1a197ca35cf/scheduler/scheduler_system.go#L442

Reply:
Yes, thank you!